src/test/java/cruise/umple/sample/downloader/ConsoleMainTest.java

```diff
@@ -17,19 +14,16 @@
 import com.google.inject.Inject;
 import com.google.inject.Provider;
 
-import cruise.umple.sample.downloader.ConsoleMain.ImportRuntimeData;
+import cruise.umple.sample.downloader.consistent.ImportRepository;
+import cruise.umple.sample.downloader.consistent.ImportRepositorySet;
 import cruise.umple.sample.downloader.util.MockDocumentFactoryModule;
-import cruise.umple.sample.downloader.util.Pair;
 
 /**
  * Created by kevin on 15-02-23.
  */
-@Guice(modules={DownloaderModule.class, MockDocumentFactoryModule.class})
+@Guice(modules={MockDocumentFactoryModule.class})
```

Comment:
I'm not sure what you mean as RTFM? As far as I know that phrase([RTFM](http://en.wikipedia.org/wiki/RTFM)) and I'm not sure it applies here?

I'm unsure what the approach should be for annotations. Only some languages support them. For example, PHP supports them with an [additional library](https://github.com/php-annotations/php-annotations), Java fully does, Python/Ruby do, and C++ has [very limited  support](http://en.cppreference.com/w/cpp/language/attributes) (not user customizable). 

Seems like the approach might be to have a flag to turn them on and have it off by default. Warn if they're encountered while the flag is off.